### DISCUSSION TITLE
argo-workflows/service-accounts/

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#11809] [General]
# argo-workflows/service-accounts/

https://argoproj.github.io/argo-workflows/service-accounts/

**Comment by muratov-aleksei** [comment]
Hi is it possible to pass SA assigned to a user to run workflow? Right now it seems SA mentioned in Workflow/WorkflowTemplate are hard coded and making it impossible differentiate access per user.
